app/lib/constants/role.rb

```diff
@@ -1,27 +1,33 @@
 module Constants
   module Role
-    BASE_ROLES = ["Warn",
-                  "Comment Suspend",
-                  "Suspend",
-                  "Regular Member",
-                  "Trusted",
-                  "Pro"].freeze
+    BASE_ROLES = [
+      "Warn",
+      "Comment Suspend",
+      "Suspend",
+      "Regular Member",
+      "Trusted",
+      "Pro",
```

Comment:
now that I saw it in the diff, I wonder, is `Pro` still a thing?

Reply:
I think we technically still have the functionality behind the `Pro` role 😅

Reply:
Ah yeah, the analytics! I thought we renamed that to `analytics` for some reason